src/components/BaseCard.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="flex flex-col rounded-lg shadow-md overflow-hidden">
+    <div class="flex-shrink-0">
+      <img class="h-48 w-full object-cover" :src="post.attributes.src || 'https://i.stack.imgur.com/y9DpT.jpg'">
+    </div>
+    <div class="flex-1 bg-white p-6 flex flex-col justify-between">
+      <div class="flex-1">
+        <p class="text-sm font-medium text-indigo-600">
+          <span
+              v-for="(tag, index) in post.attributes.tags || []"
+              :key="index"
+              class="hover:underline mr-2">{{ tag.name }}</span>
+        </p>
+        <router-link :to="'blogs/' + post.attributes.slug" class="block mt-2">
```

Comment:
use newer syntax string interpolation here, `blogs/${post.attributes.slug}` should the same thing